sdk/monitor/azure-monitor-opentelemetry-exporter/src/main/java/com/azure/monitor/opentelemetry/exporter/AzureMonitorInstaller.java

```diff
@@ -0,0 +1,113 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.monitor.opentelemetry.exporter;
+
+import com.azure.monitor.opentelemetry.exporter.implementation.AzureMonitorExporterProviderKeys;
+import com.azure.monitor.opentelemetry.exporter.implementation.AzureMonitorLogRecordExporterProvider;
+import com.azure.monitor.opentelemetry.exporter.implementation.AzureMonitorMetricExporterProvider;
+import com.azure.monitor.opentelemetry.exporter.implementation.AzureMonitorSpanExporterProvider;
+import io.opentelemetry.sdk.autoconfigure.AutoConfiguredOpenTelemetrySdkBuilder;
+import io.opentelemetry.sdk.metrics.Aggregation;
+import io.opentelemetry.sdk.metrics.InstrumentSelector;
+import io.opentelemetry.sdk.metrics.View;
+
+import java.util.HashMap;
+import java.util.Map;
+import java.util.function.Consumer;
+
+/**
+ * Use {@link AutoConfiguredOpenTelemetrySdkBuilder} to install the azure monitor exporters.
+ */
+public final class AzureMonitorInstaller {
```

Comment:
nit: consider adding `@Fluent` annotation, it helps to verify that fluent pattern is implemented properly